src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md

```diff
@@ -83,6 +84,29 @@ Template          : iotc-default@1.0.0
 SubscriptionId    : XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
 ResourceGroupName : MyResourceGroupName
 
+### Example 3 Update App Sku Info
+```powershell
+PS C:\> Set-AzureRmIotCentralApp -ResourceGroupName "MyResourceGroupName" -Name "MyAppResourceName" -Sku "ST2"
+```
+
+Update the sku on an existing IoT Central Application.
+
+Example Output:
+
+ResourceId        : /subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/MyResourceGroupName/providers/Microsoft
+                    .IoTCentral/IoTApps/MyAppResourceName
+Name              : MyAppResourceName
+Type              : Microsoft.IoTCentral/IoTApps
+Location          : westus
+Tag               : 
+Sku               : Microsoft.Azure.Commands.IotCentral.Models.PSIotCentralAppSkuInfo
+ApplicationId     : XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+DisplayName       : Display Name
+Subdomain         : MyAppSubdomain
+Template          : iotc-default@1.0.0
```

Comment:
should we use iotc-pnp-preview instead?

Reply:
Let's leave it as is for this PR. We can go through the documentation later.

Reply:
sounds good!